src/Hook/DisplayBackOfficeHeader.php

```diff
@@ -29,11 +30,24 @@ class DisplayBackOfficeHeader extends Hook
      */
     public function execute(array $params = [])
     {
+        if (defined('_PS_VERSION_')
+            && version_compare(_PS_VERSION_, '8', '>=')) {
+            try {
+                $this->module->getOauth2Middleware()->execute();
+            } catch (IdentityProviderException $e) {
+                $this->logger->error('error while executing middleware : ' . $e->getMessage());
```

Comment:
That would be interresting to know here that the error is fomr IdentityProviderException